lang/talon.talon

```diff
@@ -20,6 +20,17 @@ insert:
 key:
 	insert('key()')
 	edit.left()
+control key:
```

Comment:
wondering if we should use the modifiers capture here, so the same words are used?

`<user.modifiers> key: "{user.modifiers}"`

Reply:
Yes, thanks for showing some more good Talon